js/addToTask.js

```diff
@@ -0,0 +1,74 @@
+import {
+  createTaskButton,
+  content,
+  list,
+  inputTask,
+  input,
+  addTask,
+  trashInput,
+  today,
+  COMPLETE,
+  EDIT,
+  DELET,
+  CLICK_EVENT,
+  HIDE,
+} from "/js/constants.js"; //named import
+
+export let LIST = [];
+export let id = 0;
+let tarikh;
+function addToDo(toDo, id, done, edit, trash) {
+  if (trash) {
+    return;
+  }
```

Comment:
Can you please explain this, what does trash means?

Reply:
here trash is a property, I have added this property along with other properties (id, done, edit, etc) to track the actions performed on tasks. But these lines of code are unnecessary since I am always passing trash false from the addTask event listener. in my latest push, I have removed these lines,